build.sbt

```diff
@@ -292,6 +292,14 @@ lazy val marathon = (project in file("."))
   .settings(formatSettings: _*)
   .settings(packagingSettings: _*)
   .settings(
+    version := {
+      val v = sys.env.get("MARATHON_VERSION").filter(_ != "git").getOrElse(version.value)
```

Comment:
Ah, got it. I should have read on. Do we want to allow this? For 1.6 onward we do not require this, do we?

Reply:
I think it is nice, for 1.5. I have a preference to not create a tag until we are sure that the packages built past tests, the integration tests pass, etc. It avoids the chicken and egg scenario.